client/src/styles/_card.scss

```diff
@@ -0,0 +1,3 @@
+.ant-card-bordered {
+  border-radius: 10px;
```

Comment:
This changes the radius of the stats cards from 16px to 10px and creates a design bug. Make sure you don't break the cards 😅 

<img width="380" alt="Screenshot 2021-08-22 at 16 51 56" src="https://user-images.githubusercontent.com/2820084/130357652-5e4b9623-a682-4a4b-bea7-4843fafbc997.png">

Reply:
good catch 👍  fixed